### PR TITLE
WIP: dev-adrian01c welcome section

### PR DESCRIPTION
**Pull request by Adrian01C**

### CODE REVIEW

File: web/index.html [hunk 1]
```diff
@@ -2,10 +2,28 @@
...
-    <p>I am the file <code>index.html</code> and you want to change me!</p>
+    <div class="about-me">
+      <h1>Virgil A. Ciociu</h1>
+      <h2>Starter Developer</h2>
+      <h3>About me</h3>
+      <div class="box-shadow">
+        <p class="text-description">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Modi dolores laborum tempora illum eligendi eos voluptatum. Aspernatur consectetur aliquam sed nostrum? Cupiditate excepturi iste facilis nulla eaque autem, veniam perspiciatis?</p.>
+      </div>
+      <button>Download my CV</button>
+      <button .class="disc">
```
Comment (tandaraelena) [inline web/index.html:18]: Please delete the "." before the class word.

File: web/index.html [hunk 2]
```diff
@@ -2,10 +2,28 @@
...
+
```
Comment (tandaraelena) [inline web/index.html:19]: Please delete this empty line

**Review by tandaraelena — CHANGES_REQUESTED** [review]

File: web/index.html [hunk 3]
```diff
@@ -2,10 +2,28 @@
 <html lang="en">
   <head>
     <title>My CV v2 - starter</title>
+    <link rel="stylesheet" href="<link rel="preconnect" href="https://fonts.gstatic.com">
```
Comment (tandaraelena) [inline web/index.html:5]: This line is confusing. I think you have an extra href attribute.

**Review by tandaraelena — CHANGES_REQUESTED** [review]

**Review by psyhedeliq — APPROVED** [review]
🚀 🚀 🚀

**Review by simonamarcu2 — APPROVED** [review]
good work

**Review by RalucaRbc — APPROVED** [review]

**Review by tandaraelena — APPROVED** [review]

**Review by biggicode — APPROVED** [review]
I approve it but you should change what Elena said

File: web/css/style.css [hunk 4]
```diff
@@ -3,6 +3,11 @@ body {
   font-family: Roboto;
 }
 
+grid {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
```
Comment (ValentinaVanu) [inline web/css/style.css:8]: Can also be written as:
```suggestion
  grid-template-columns: repeat(4, 1fr);
```

**Review by ValentinaVanu — APPROVED** [review]

**Review by simonamarcu2 — APPROVED** [review]
💯

**Review by vasilemaftita — APPROVED** [review]
Nice.

**Review by vasilemaftita — APPROVED** [review]

**Review by Constantin-Matei — APPROVED** [review]

**Review by vasilemaftita — APPROVED** [review]
Nice; Work.

**Review by marianzburlea — APPROVED** [review]

**Pull request merged**